Bottom: c37fc2e23559aefd4fc97a627df1600860ce5381
Top:    175706f4811327b820410ad8fb9caf4d57be4d0b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:56:09 -0500

Refresh of all-docker-image-repositories

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index e25c4a7..97c0a9b 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh {{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/restore.date.txt'
@@ -76,6 +73,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mantisbt'
 
+- name: copy script
+  copy:
+    remote_src: true
+    src: '{{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh'
+    dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
+
 - name: build image
   docker_image:
     name: '{{ mantisbt_image_name }}'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index e16bb78..f084ca6 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}/restore.date.txt'
@@ -77,6 +74,9 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mediawiki'
 
+- name: copy script
+  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
+
 - name: build image
   docker_image:
     name: '{{ wiki_image_name }}'
